frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/scrolling/styled_scroll_bar.dart

```diff
@@ -225,6 +225,7 @@ class ScrollbarListStack extends StatelessWidget {
   final EdgeInsets? scrollbarPadding;
   final Color? handleColor;
   final Color? trackColor;
+  final bool showTrack;
```

Comment:
You added this property, but don't set it to `true` anywhere, and it defaults to `false` inside `StyledScrollbar`, do we need it?

Reply:
I added it because `ScrollbarListStack` has `showTrack` parameter, but it didn't pass down to `StyledScrollbar` in the previous code. The previous UI didn't show track, so I didn't set `true` there. 
However, I think it would be better to show the track there. 
I also updated the default color of track like this: 
Light mode: 
<img width="148" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/14248245/8f03f9fc-4e19-4f96-8242-d10d810d7e59">

Dark mode:
<img width="133" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/14248245/4346050e-9c9b-4930-accc-8730a797f9d4">